### PR TITLE
Base shape and draw menu instruction

### PR DESCRIPTION
**Pull request by IvanHalych**

### CODE REVIEW

File: DrawingConsole/Shape/Circle.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        {
+            X = x;
+            R = r;
+        }
+
+        public Point X { get; }
+        public double R { get; }
+        public double Square
+        {
+            get { return Math.PI * Math.PI * R;  }
```
Comment (yevIbrahimov) [inline DrawingConsole/Shape/Circle.cs:23]: possible use Math.Pow

**Review by Kirill-bab — APPROVED** [review]
гуд

**Review by yevIbrahimov — APPROVED** [review]
good

File: DrawingConsole/Shape/Circle.cs (hunk 1 above)
Comment (IvanHalych) [reply DrawingConsole/Shape/Circle.cs:23]: ok

**Review by IvanHalych — COMMENTED** [review]
ok

**Review by yevIbrahimov — APPROVED** [review]
Nice code

**Review by Kirill-bab — APPROVED** [review]
Cool

**Pull request merged**